circus-train-core/src/main/java/com/hotels/bdp/circustrain/core/replica/Replica.java

```diff
@@ -30,6 +30,8 @@
 import java.util.Map;
 import java.util.Objects;
 
+import com.google.common.annotations.VisibleForTesting;
```

Comment:
Minor, but these imports should go along with the other google imports. I've made a mental note to update our hotels-oss-parent with a plugin that automatically does this that we've started using in EG org on GitHub so this is automated by Maven.

Reply:
Yeah, I am struggling with the correct IntelliJ settings to match circus train's preferred import layout.  Obviously DSP standards are not matching up well with HCOM in this circumstance. Do you have a doc or a settings file or something so that I can update?

Reply:
This will do it once I've merged and released the parent - https://github.com/HotelsDotCom/circus-train/pull/168/files

I'm getting timeouts publishing to Maven Central right now, they've had lots of scalability issues in the last few months so I'm going to leave it for today and try again tomorrow.